holmes/core/config.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+config_path_dir: str = os.environ.get(
+    "CONFIG_PATH_DIR", os.path.expanduser("~/.holmes")
```

Comment:
Can you make it HOLMESGPT_CONFIGPATH_DIR?

The current value is,too generic and could conflict with some other cli tool